src/locutus/model/terminology.py

```diff
@@ -160,7 +164,17 @@ def build_code_list(mapping):
         code_id = mapping["code"]
 
         for coding in mapping["codes"]:
-            codes.append(Coding(**coding))
+            # Get the 'valid' field separately, defaulting to None if not found
+            valid = coding.get('valid', None)

```

Comment:
If the valid defaults to True when missing, we should just set that second parameter to True.